pages/common/fselect.md

```diff
@@ -0,0 +1,23 @@
+# fselect
+
+> Find files with SQL-like queries.
+
+- Find temporary or config files (full path and size):
```

Comment:
How about - "Select full path and size from temporary or config files in a given directory."

Reply:
Sure, why not :)